proxy/src/serverless/sql_over_http.rs

```diff
@@ -72,6 +75,14 @@ static TXN_DEFERRABLE: HeaderName = HeaderName::from_static("neon-batch-deferrab
 
 static HEADER_VALUE_TRUE: HeaderValue = HeaderValue::from_static("true");
 
+fn bytes_to_pg_text<'de, D>(deserializer: D) -> Result<Vec<Option<String>>, D::Error>
+where
+    D: serde::de::Deserializer<'de>,
+{
+    let json: Vec<Value> = serde::de::Deserialize::deserialize(deserializer)?;
+    Ok(json_to_pg_text(json))
```

Comment:
I wonder if we can avoid this intermediate vec. maybe with https://docs.rs/serde_with/3.6.0/serde_with/#advanced-serde_as-usage

Reply:
Hm, it doesn't look like something trivial. Added TODO.